components/script/dom/node.rs

```diff
@@ -2922,7 +2935,9 @@ impl Into<LayoutNodeType> for NodeTypeId {
     fn into(self) -> LayoutNodeType {
         match self {
             NodeTypeId::Element(e) => LayoutNodeType::Element(e.into()),
-            NodeTypeId::CharacterData(CharacterDataTypeId::Text) => LayoutNodeType::Text,
+            NodeTypeId::CharacterData(CharacterDataTypeId::Text(TextTypeId::Text)) => {
```

Comment:
I feel like this should also be an underscore? We should be rendering CData nodes, yes?